docs/local-library-setup.md

```diff
@@ -0,0 +1,80 @@
+---
+id: local-library-setup
+title: Local libraries setup
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+The local library is created outside of the `android/` and `ios/` folders and makes use of autolinking to integrate with your app. To create local library we will use [create-react-native-library](https://callstack.github.io/react-native-builder-bob/create). This tool contains all the necessary templates.
+
+### Getting Started
+
+Inside your React Native application's root folder, run the following command:
+
+```shell
+npx create-react-native-library@latest awesome-module
+```
+
+Where `awesome-module` is the name you would like for the new module. After going through the prompts, you will have a new folder called `modules` in your project's root directory which contains the new module.
+
+### Linking
+
+By default, the generated library is automatically linked to the project using `link:` protocol when using Yarn and `file:` when using npm:
+
+<Tabs groupId="package-manager" queryString defaultValue={constants.defaultPackageManager} values={constants.packageManagers}>
+
+<TabItem value="npm">
+
+```json
+"dependencies": {
+  "awesome-module": "file:./modules/awesome-module"
+}
+```
+
+</TabItem>
+<TabItem value="yarn">
+
+```json
+"dependencies": {
+  "awesome-module": "link:./modules/awesome-module"
+}
+```
+
+</TabItem>
+</Tabs>
+
+This creates a symlink to the library under `node_modules` which makes autolinking work.
+
+### Installing dependencies
+
+:::note
+Depending on module type you may have to make additional steps to make it work. Follow the instructions provided by the tool.
+:::
+
+To link the module you need to install dependencies:
+
+<Tabs groupId="package-manager" queryString defaultValue={constants.defaultPackageManager} values={constants.packageManagers}>
+
+<TabItem value="npm">
+
+```shell
+npm install
+```
```

Comment:
Should we make a call and only use `yarn`, @cortinico?
I remember having discussions that npm is brittle, but I'm, not totally sure.

Reply:
In theory this should be tabbed npm/yarn as we do for all the other pages (like here https://reactnative.dev/docs/react-devtools)